cpu/stm32/stmclk/stmclk_mp1.c

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright (C) 2020 Savoir-faire Linux
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     cpu_stm32
+ * @{
+ *
+ * @file
+ * @brief       Implementation of STM32 clock configuration for STM32MP1
+ *
+ * @author      Gilles DOFFE <gilles.doffe@savoirfairelinux.com>
+ *
+ * @}
+ */
+
+#include "cpu.h"
+#include "stmclk.h"
+#include "periph_conf.h"
+
+#if IS_USED(MODULE_STM32MP1_ENG_MODE)
```

Comment:
Is it possible to include this file is `stm32mp1_eng_mode` module is used ? If yes, then this `#if` is not needed since all this could be handled in the stmclk Makefile.